README.md

```diff
@@ -55,6 +62,35 @@ $rollout = $this->getServiceLocator()->get('zf2_rollout');
 
 Refer to the documentation of [opensoft/rollout](https://github.com/opensoft/rollout) for more information on how to use the library.
 
+# Description
+
+If you want to display the description of a feature in your views. You need to
```

Comment:
```suggestion
Since the Rollout library doesn't have (yet) a functionality to describe its feature flags, you can define them through this module. To do so, simply add your feature flag identifier in the 'features' sections of the rollout configuration as shown here:
```

Reply:
Actually the Ruby version already supports the description. You can set the description on an existing feature.